applications/fishing-map/src/features/workspace/shared/NewWorkspaceModal.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from 'react'
+import { useSelector } from 'react-redux'
+import { event as uaEvent } from 'react-ga'
+import { useTranslation } from 'react-i18next'
+import InputText from '@globalfishingwatch/ui-components/dist/input-text'
+import Button from '@globalfishingwatch/ui-components/dist/button'
+import { getOceanAreaName, OceanAreaLocale } from '@globalfishingwatch/ocean-areas'
+import Modal from '@globalfishingwatch/ui-components/dist/modal'
+import Radio from '@globalfishingwatch/ui-components/dist/radio'
+import { saveCurrentWorkspaceThunk } from 'features/workspace/workspace.slice'
+import { selectWorkspace } from 'features/workspace/workspace.selectors'
+import { DEFAULT_WORKSPACE_ID } from 'data/workspaces'
+import { useAppDispatch } from 'features/app/app.hooks'
+import { pickDateFormatByRange } from 'features/map/controls/MapInfo'
+import { formatI18nDate } from 'features/i18n/i18nDate'
+import { selectViewport } from 'features/app/app.selectors'
+import { useTimerangeConnect } from 'features/timebar/timebar.hooks'
+import styles from './NewWorkspaceModal.module.css'
+
+type NewWorkspaceModalProps = {
+  isOpen: boolean
+  onClose: () => void
+  onCreate?: () => void
+}
+
+function NewWorkspaceModal({ isOpen, onClose, onCreate }: NewWorkspaceModalProps) {
+  const [name, setName] = useState('')
+  const [createAsPublic, setCreateAsPublic] = useState(true)
+  const [loading, setLoading] = useState(false)
+  const { t, i18n } = useTranslation()
+  const dispatch = useAppDispatch()
+  const viewport = useSelector(selectViewport)
+  const timerange = useTimerangeConnect()
+  const workspace = useSelector(selectWorkspace)
+
+  useEffect(() => {
+    if (isOpen) {
+      const isDefaultWorkspace = workspace?.id === DEFAULT_WORKSPACE_ID
+      const areaName = getOceanAreaName(viewport, { locale: i18n.language as OceanAreaLocale })
+      const dateFormat = pickDateFormatByRange(timerange.start as string, timerange.end as string)
+      const start = formatI18nDate(timerange.start as string, {
+        format: dateFormat,
+      })
+        .replace(',', '')
+        .replace('.', '')
+      const end = formatI18nDate(timerange.end as string, {
+        format: dateFormat,
+      })
+        .replace(',', '')
+        .replace('.', '')
```

Comment:
```suggestion
      const formatTimerangeBoundary = (boundary?: string) => {
        return formatI18nDate(boundary as string, {
        format: dateFormat,
      })
        .replace(/[\.\,]/g, '')
      }
      const start = formatTimerangeBoundary(timerange.start)
      const end = formatTimerangeBoundary(timerange.end)
```

Reply:
Thanks, added in https://github.com/GlobalFishingWatch/frontend/commit/264d9caefaa5e320a92028293a3f2e1b6d1adb75